src/iarc7_motion/linear_motion_profile_generator.py

```diff
@@ -9,121 +9,142 @@
 
 from iarc7_msgs.msg import MotionPointStamped, MotionPointStampedArray
 
+
```

Comment:
Why so much whitespace in this file? also, why was the formatting changed below? it was just fine before....